saw-remote-api/scripts/test_docker.sh

```diff
@@ -44,6 +45,7 @@ popd
 echo "killing saw-remote-api docker container"
 
 docker container kill saw-remote-api
+docker container rm saw-remote-api
```

Comment:
These two cleanup lines will only be run when the test is successful, but not when it fails for some reason.  This may not be as important when these are done in a CI environment that discards the entire CI container at the end of the run, but to avoid this for situations where that's not the case, I'd recommend removing these and adding to the beginning of the file:
```
cleanup () {
  docker container kill saw-remote-api || true
  docker container rm saw-remote-api || true
}
trap cleanup EXIT
```

Reply:
I guess you'd want to move the preceding `echo` as well..

Reply:
Ah, good idea!